lambdas/accesstoken/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+	id "java"
+	id 'io.freefair.aspectj.post-compile-weaving' version '6.3.0'
+}
+
+dependencies {
+	implementation project(":lib"),
+			configurations.lambda,
+			configurations.nimbus,
+			configurations.gson,
+			configurations.jackson
+
+	aspect configurations.powertools
+
+	testImplementation configurations.tests
+	testRuntimeOnly configurations.test_runtime
+}
+
+task buildZip(type: Zip) {
```

Comment:
Do we need this task because we dont use it in the deployment [here](https://github.com/alphagov/di-ipv-cri-address-api/blob/4969c03c3c29ea9300fbce9a20ebbd5c5f28a6e9/.github/workflows/deploy.yml#L44)

Reply:
the `buildZip` task is used in the block below, see `build.finalizedBy(buildZip)`